messages/sfdxurl.store.md

```diff
@@ -22,8 +22,16 @@ You can also create a JSON file that has a top-level property named sfdxAuthUrl
 
 Path to a file that contains the Salesforce DX authorization URL.
 
+# flags.sfdx-url.summary
+
+The Salesforce DX authorization URL as a string.
```

Comment:
If we add this flag, the command summary and description will also need to be updated because at the moment it assumes that you can specify the URL only via a file.

Reply:
I committed this change and also updated the description to remove the assumptions about the file. Hopefully it's clear!